### DISCUSSION TITLE
NavigationStack Introspect missing

### DISCUSSION THREAD

**Discussion by Fil0611** [siteline/SwiftUI-Introspect#224] [Q&A] [answered]
NavigationView is now deprecated.
Could you add an introspect for NavigationStack?

**Comment by davdroman** [comment]
Even though it may not seem obvious, you can still use `.introspectNavigationController` on a `NavigationStack` because the underlying component is still a `UINavigationController`, just like with `NavigationView`.

Moving this to a discussion since it's more a question than a feature request.

**Reply by davdroman** [reply, marked as answer]
While my answer above still holds, with the newly released `SwiftUIIntrospect` module you can now do:

```swift
NavigationStack {
    ...
}
.introspect(.navigationStack, on: .iOS(.v16), .tvOS(.v16)) { navigationController in
    ...
}
```
